R/api.R

```diff
@@ -16,109 +16,379 @@
   get_content(hn_api_response)
 }
 
-#' Retrieves the item corresponding to the specified id
+
+#' @title Print for "hackerlist" type objects
+#'
+#' @param x "hackerlist" type list
+#'
+#' @param ... further arguments passed to or from other methods
+#'
+#' @export
+#'
+#' @examples
+#'
+#' hacker_news <- get_top_stories(max_items = 5)
+#' print(hacker_news)
+
+print.hackerlist <- function(x, ...) {
```

Comment:
Please keep a common convention when naming classes. E.g. we already have a class called `hn_api_response`. Why not call this class `hn_list` then? :)